### PR TITLE
Update outdated rollbar gem

### PR DESCRIPTION
**Pull request by ruslankhaertdinov**

### CODE REVIEW

**Comment by ruslankhaertdinov** [comment]
@timurvafin please take a look

**Comment by timurvafin** [comment]
@ruslankhaertdinov do we need update rollbar initializer as well?

**Comment by ruslankhaertdinov** [comment]
@timurvafin no, there are no mandatory changes in the initializer file: https://github.com/rollbar/rollbar-gem#rails

**Comment by ruslankhaertdinov** [comment]
@timurvafin but we can store it anyway, because it has brief documentation of usage. Give me 5 mins.

**Comment by ruslankhaertdinov** [comment]
@timurvafin done

File: Gemfile [hunk 1]
```diff
@@ -18,6 +18,7 @@ gem "interactor"
 gem "kaminari"
 gem "rack-cors", require: "rack/cors"
 gem "responders"
+gem "rollbar"
```
Comment (timurvafin) [inline Gemfile]: What is the reason to have rollbar required for all envs?

Comment (ruslankhaertdinov) [inline Gemfile]: We may have explicit use `Rollbar` in the app, e.g. `Rollbar.debug("some info", user_id: current_user.id)` and in this case we should have `Rollbar` for each env (like in the [rails-base](https://github.com/fs/rails-base/blob/master/Gemfile#L41) template)

**Pull request merged**